unittest/wrapper/ReadWriteUnittest.cpp

```diff
@@ -13,9 +13,9 @@ class ReadWriteUnittest: public ::testing::Test {
 
     // These vectors are used as Fortran-style storage
     std::vector<int> fieldInt2, fieldInt3;
-    std::vector<Real> fieldReal1, fieldReal3;
+    std::vector<double> fieldReal1, fieldReal3;
     std::vector<int> fieldcheckInt2, fieldcheckInt3;
-    std::vector<Real> fieldcheckReal1, fieldcheckReal3;
+    std::vector<double> fieldcheckReal1, fieldcheckReal3;
```

Comment:
yes, before we were implicitly testing single precision with the matrix build, and now this is gone. 
@andyspiros could you parametrized these classes of test with a template parameter to launch double and float gtests?